infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/brokerphases/DeployBroker.java

```diff
@@ -88,6 +95,7 @@ public DeployBroker(
       deployments.create(pluginBrokerPod);
 
       LOG.debug("Brokers pod is created for workspace '{}'", workspaceId);
+      finishSpanIfExists(tracingSpan);
```

Comment:
If a case of an exception it could be bypassed. Should we move it to finally section?

Reply:
Avoiding this would have to be done in a catch block instead of finally, otherwise the span would not complete until all remaining phases would be done (since finally won't execute until `nextPhase.execute()` is complete. 

I've added a catch block here, but it's more messy (catch exception, do something, rethrow exception)